src/build_context.rs

```diff
@@ -251,23 +249,101 @@ impl BuildContext {
         python_interpreter: Option<&PythonInterpreter>,
         artifact: &Path,
         platform_tag: Option<PlatformTag>,
-    ) -> Result<Policy> {
+    ) -> Result<(Policy, Vec<Library>)> {
         if self.skip_auditwheel {
-            return Ok(Policy::default());
+            return Ok((Policy::default(), Vec::new()));
         }
 
         let target = python_interpreter
             .map(|x| &x.target)
             .unwrap_or(&self.target);
 
-        let policy = auditwheel_rs(artifact, target, platform_tag).context(
-            if let Some(platform_tag) = platform_tag {
-                format!("Error ensuring {} compliance", platform_tag)
+        let (policy, should_repair) =
+            auditwheel_rs(artifact, target, platform_tag).with_context(|| {
+                if let Some(platform_tag) = platform_tag {
+                    format!("Error ensuring {} compliance", platform_tag)
+                } else {
+                    "Error checking for manylinux/musllinux compliance".to_string()
+                }
+            })?;
+        let external_libs = if should_repair && !self.editable {
```

Comment:
Is this even executed for editable installs?

Reply:
I shouldn't, I was been defensive there.